app/components/emoji_picker/emoji_picker_row.js

```diff
@@ -20,7 +21,7 @@ export default class EmojiPickerRow extends Component {
     }
 
     shouldComponentUpdate(nextProps) {
-        return this.props.items.length !== nextProps.items.length;
+        return !shallowEqual(this.props.items, nextProps.items);
```

Comment:
Does checking this shallowly improve performance by making previous rows rerender less when loading new emojis or is it possibly worse do to shallowly checking constantly

Reply:
Actually, this is probably better. It's not connected to the store, so this isn't being called anywhere near as often as it would otherwise